scripts/clear_chronos_user_id_and_submissions.py

```diff
@@ -7,15 +7,24 @@
 django.setup()
 
 from osf import models
+from website.app import init_app
 
 logger = logging.getLogger(__name__)
-logging.basicConfig(level=logging.INFO)
-
 
 def main(dry_run=True):
-    models.OSFUser.objects.all().update(chronos_user_id=None)
-    models.ChronosSubmission.objects.all().delete()
+    if dry_run:
+        for user in models.OSFUser.objects.all():
```

Comment:
This list would be too long to be useful, I'd just log users that actually had a `chronos_user_id` to begin with.